tool/vfx/vfxVkSection.h

```diff
@@ -452,7 +453,7 @@ class SectionGraphicsState : public Section {
 
 private:
   SectionNggState m_nggState;
-  static const unsigned MemberCount = 28;
+  static const unsigned MemberCount = 25;
```

Comment:
I can see one new member added, so I'd expect this to change to **increase** to 29 instead of **decreasing** to 25.

Reply:
we removed several members before in internal branch, but didn't update the count, the change to 25 is for that, this count is good for open source  too.

Reply:
Is this compatible with old XGL/PAL code drops? We are still on v215.

Reply:
this change is independent of xgl/pal, there should be no compatible issue.